sys-libs/libcpuid/metadata.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM 'http://www.gentoo.org/dtd/metadata.dtd'>
+<pkgmetadata>
+<maintainer type='person'>
+	<email>beatussum@protonmail.com</email>
+	<name>Mattéo Rossillol</name>
+</maintainer>
```

Comment:
You need to add a project or a Gentoo developer to commit for you. Usually it's the proxy-maint project. Check other ebuilds for example how it's added here.